### PR TITLE
[testing_codelab]: Update runners

### PR DESCRIPTION
**Pull request by domesticmouse**
Hey @abd99, are you up for a code review? =)

### CODE REVIEW

**Comment by abd99** [comment]
Yes, I am.

File: testing_codelab/step_03/pubspec.yaml [hunk 1]
```diff
@@ -16,16 +16,16 @@ dependencies:
...
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flutter_driver: # new
-    sdk: flutter # new
-  integration_test: # new
-    sdk: flutter # new
-  test: ^1.14.4 # new
+  flutter_driver:
```
Comment (abd99) [inline testing_codelab/step_03/pubspec.yaml:24]: Do we need the `# new` comments?

**Review by abd99 — APPROVED** [review]
lgtm

File: testing_codelab/step_03/pubspec.yaml (hunk 1 above)
Comment (domesticmouse) [reply testing_codelab/step_03/pubspec.yaml:24]: Probably not, thanks!

Comment (domesticmouse) [reply testing_codelab/step_03/pubspec.yaml]: Added `# new` comments back

**Pull request merged**